cpu/fe310/include/cpu_conf.h

```diff
@@ -19,35 +19,15 @@
 #ifndef CPU_CONF_H
 #define CPU_CONF_H
 
-/**
- * @name Configuration of default stack sizes
- * @{
- */
-#ifndef THREAD_EXTRA_STACKSIZE_PRINTF
-#define THREAD_EXTRA_STACKSIZE_PRINTF   (256)
-#endif
-#ifndef THREAD_STACKSIZE_DEFAULT
-#define THREAD_STACKSIZE_DEFAULT        (1024)
-#endif
-#ifndef THREAD_STACKSIZE_IDLE
-#define THREAD_STACKSIZE_IDLE           (256)
-#endif
-/** @} */
-
-/**
- * @brief   Declare the heap_stats function as available
- */
-#define HAVE_HEAP_STATS
-
-/**
- * @brief   This arch uses the inlined irq API.
- */
-#define IRQ_API_INLINED     (1)
+#include "cpu_conf_common.h"
+#include "vendor/platform.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+#define CLINT_BASE_ADDR     (CLINT_CTRL_ADDR)
```

Comment:
I think this define should be documented

Reply:
I've added a bit of doxygen. More extensive documentation is available in the `periph/coretimer.c` file